src/mono/mono/mini/aot-compiler.c

```diff
@@ -5303,6 +5306,8 @@ MONO_RESTORE_WARNING
 						export_name = (char *)g_malloc (slen + 1);
 						sprintf (export_name, "%s%s", acfg->user_symbol_prefix, named);
 						export_name [slen] = 0;
+
+						g_ptr_array_add (acfg->exported_methods, method);
```

Comment:
Can it create duplicate items in an array?

Reply:
Maybe if the method has multiple unmanagedcallersonly attributes, but it shouldn't be a problem in practice.